### ISSUE TITLE
Ups, so pr_finish() can bite.

### ISSUE THREAD

**Issue by maurolepore** [r-lib/usethis#805]
Oops, I run `pr_fetch()`, committed, then run `pr_finish()` without running `pr_push()`. So I lost the PR (luckily just one line). I now think it makes sense -- should have pushed -- but then I didn't know what to expect and hadn't read the docs (oh no!). Am I the only one learning this the hard way? Would it make sense for `pr_finish()` to check against the remote and ask to confirm if the local repo is behind?

**Comment by jennybc** [comment]
Yeah, it sounds like there should be more safeguards here. I think `pr_pause()` is what you probably wanted.

It sounds like in this case, that the local repo is *ahead*, but being behind is more typical (because maintainer or someone else has updated the branch).

If it had been a more extensive piece of work that you "lost", you could retrieve it from the reflog. But I realize that no one likes going into the reflog 😬
